2019-11-15 19:31:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Unbekannter Datentyp: "INTETER"
Unknown data type: "INTETER"; SQL statement:
CREATE TABLE IF NOT EXISTS REGISTRATION (id INTETER auto increment), name VARCHAR(100), tenant VARCHAR(100), size FLOAT, PRIMARY KY (id) ) [50004-200]
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:205)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:181)
	at com.h2database@1.4.200/org.h2.command.Parser.parseColumnWithType(Parser.java:5971)
	at com.h2database@1.4.200/org.h2.command.Parser.parseColumnForTable(Parser.java:5697)
	at com.h2database@1.4.200/org.h2.command.Parser.parseTableColumnDefinition(Parser.java:8442)
	at com.h2database@1.4.200/org.h2.command.Parser.parseCreateTable(Parser.java:8379)
	at com.h2database@1.4.200/org.h2.command.Parser.parseCreate(Parser.java:6276)
	at com.h2database@1.4.200/org.h2.command.Parser.parsePrepared(Parser.java:903)
	at com.h2database@1.4.200/org.h2.command.Parser.parse(Parser.java:843)
	at com.h2database@1.4.200/org.h2.command.Parser.parse(Parser.java:815)
	at com.h2database@1.4.200/org.h2.command.Parser.prepareCommand(Parser.java:738)
	at com.h2database@1.4.200/org.h2.engine.Session.prepareLocal(Session.java:657)
	at com.h2database@1.4.200/org.h2.engine.Session.prepareCommand(Session.java:595)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:164)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:126)
	at com.utilities_statement/db.CreateTable.main(CreateTable.java:27)
2019-11-15 19:31:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "CREATE TABLE IF NOT EXISTS REGISTRATION (ID INTEGER AUTO[*] INCREMENT), NAME VARCHAR(100), TENANT VARCHAR(100), SIZE FLOAT, PRIMARY KY (ID) )"; erwartet "(, FOR, UNSIGNED, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"
Syntax error in SQL statement "CREATE TABLE IF NOT EXISTS REGISTRATION (ID INTEGER AUTO[*] INCREMENT), NAME VARCHAR(100), TENANT VARCHAR(100), SIZE FLOAT, PRIMARY KY (ID) )"; expected "(, FOR, UNSIGNED, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE IF NOT EXISTS REGISTRATION (id INTEGER auto increment), name VARCHAR(100), tenant VARCHAR(100), size FLOAT, PRIMARY KY (id) ) [42001-200]
2019-11-15 19:32:47 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "CREATE TABLE IF NOT EXISTS REGISTRATION (ID INTEGER AUTO[*] INCREMENT, NAME VARCHAR(100), TENANT VARCHAR(100), SIZE FLOAT, PRIMARY KY (ID) )"; erwartet "(, FOR, UNSIGNED, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"
Syntax error in SQL statement "CREATE TABLE IF NOT EXISTS REGISTRATION (ID INTEGER AUTO[*] INCREMENT, NAME VARCHAR(100), TENANT VARCHAR(100), SIZE FLOAT, PRIMARY KY (ID) )"; expected "(, FOR, UNSIGNED, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE IF NOT EXISTS REGISTRATION (id INTEGER auto increment, name VARCHAR(100), tenant VARCHAR(100), size FLOAT, PRIMARY KY (id) ) [42001-200]
2019-11-15 19:35:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "CREATE TABLE IF NOT EXISTS REGISTRATION (ID INTEGER AUTO[*] INCREMENT, NAME VARCHAR(100), TENANT VARCHAR(100), SIZE FLOAT, PRIMARY KEY (ID) )"; erwartet "(, FOR, UNSIGNED, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"
Syntax error in SQL statement "CREATE TABLE IF NOT EXISTS REGISTRATION (ID INTEGER AUTO[*] INCREMENT, NAME VARCHAR(100), TENANT VARCHAR(100), SIZE FLOAT, PRIMARY KEY (ID) )"; expected "(, FOR, UNSIGNED, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE IF NOT EXISTS REGISTRATION (id INTEGER auto increment, name VARCHAR(100), tenant VARCHAR(100), size FLOAT, PRIMARY KEY (id) ) [42001-200]
2019-11-15 19:38:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "CREATE TABLE IF NOT EXISTS REGISTRATION (ID INTEGER AUTO[*] INCREMENT, NAME VARCHAR(100), TENANT VARCHAR(100), SIZE FLOAT, PRIMARY KEY (ID) );"; erwartet "(, FOR, UNSIGNED, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"
Syntax error in SQL statement "CREATE TABLE IF NOT EXISTS REGISTRATION (ID INTEGER AUTO[*] INCREMENT, NAME VARCHAR(100), TENANT VARCHAR(100), SIZE FLOAT, PRIMARY KEY (ID) );"; expected "(, FOR, UNSIGNED, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE IF NOT EXISTS REGISTRATION (id INTEGER auto increment, name VARCHAR(100), tenant VARCHAR(100), size FLOAT, PRIMARY KEY (id) ); [42001-200]
2019-11-15 19:41:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "CREATE TABLE FLAT IF[*] NOT EXISTS REGISTRATION (ID INTEGER AUTO_INCREMENT, NAME VARCHAR(100), TENANT VARCHAR(100), SIZE FLOAT, PRIMARY KEY (ID) );"
Syntax error in SQL statement "CREATE TABLE FLAT IF[*] NOT EXISTS REGISTRATION (ID INTEGER AUTO_INCREMENT, NAME VARCHAR(100), TENANT VARCHAR(100), SIZE FLOAT, PRIMARY KEY (ID) );"; SQL statement:
CREATE TABLE flat IF NOT EXISTS REGISTRATION (id INTEGER AUTO_INCREMENT, name VARCHAR(100), tenant VARCHAR(100), size FLOAT, PRIMARY KEY (id) ); [42000-200]
2019-11-15 19:42:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "CREATE TABLE IF NOT EXISTS REGISTRATION FLAT[*] (ID INTEGER AUTO_INCREMENT, NAME VARCHAR(100), TENANT VARCHAR(100), SIZE FLOAT, PRIMARY KEY (ID) );"
Syntax error in SQL statement "CREATE TABLE IF NOT EXISTS REGISTRATION FLAT[*] (ID INTEGER AUTO_INCREMENT, NAME VARCHAR(100), TENANT VARCHAR(100), SIZE FLOAT, PRIMARY KEY (ID) );"; SQL statement:
CREATE TABLE IF NOT EXISTS REGISTRATION flat (id INTEGER AUTO_INCREMENT, name VARCHAR(100), tenant VARCHAR(100), size FLOAT, PRIMARY KEY (id) ); [42000-200]
2019-11-15 19:45:45 database: wrong user or password; user: "ROOT"
org.h2.message.DbException: Falscher Benutzer Name oder Passwort
Wrong user name or password [28000-200]
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.engine.Engine.openSession(Engine.java:131)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:309)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:289)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:785)
	at org.h2.server.web.WebApp.test(WebApp.java:947)
	at org.h2.server.web.WebApp.process(WebApp.java:238)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:173)
	at org.h2.server.web.WebThread.process(WebThread.java:139)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Falscher Benutzer Name oder Passwort
Wrong user name or password [28000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:461)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 21 more
2019-11-15 19:45:50 database: wrong user or password; user: "ROOT"
org.h2.message.DbException: Falscher Benutzer Name oder Passwort
Wrong user name or password [28000-200]
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.engine.Engine.openSession(Engine.java:131)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:309)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:289)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:785)
	at org.h2.server.web.WebApp.login(WebApp.java:1008)
	at org.h2.server.web.WebApp.process(WebApp.java:223)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:173)
	at org.h2.server.web.WebThread.process(WebThread.java:139)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: Falscher Benutzer Name oder Passwort
Wrong user name or password [28000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:461)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 21 more
2019-11-15 20:05:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Datenumwandlungsfehler beim Umwandeln von "tenant1"
Data conversion error converting "tenant1" [22018-200]
2019-11-15 20:12:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabelle "FLAT" nicht gefunden
Table "FLAT" not found; SQL statement:
SELECT * FROM flat [42102-200]
2019-11-15 21:34:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Anzahl der Felder stimmt nicht überein
Column count does not match; SQL statement:
INSERT INTO Flat(description, tenant, numResidents, space, rent, movingDate, isRented) values('Geschaeftsraum klein', 'tenant1', 1, 15.0, '2011-01-01', TRUE) [21002-200]
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:205)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:181)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:170)
	at com.h2database@1.4.200/org.h2.command.dml.Insert.prepare(Insert.java:319)
	at com.h2database@1.4.200/org.h2.command.Parser.prepareCommand(Parser.java:744)
	at com.h2database@1.4.200/org.h2.engine.Session.prepareLocal(Session.java:657)
	at com.h2database@1.4.200/org.h2.engine.Session.prepareCommand(Session.java:595)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:164)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:126)
	at com.utilities_statement/db.Insert.main(Insert.java:23)
2019-11-15 21:40:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "INSERT INTO FLAT(DESCRIPTION, TENANT, NUMRESIDENTS, SPACE, RENT, MOVINGDATE, ISRENTED) VALUES('Geschaeftsraum klein', 'tenant1', 1, 15.0, 200.0, '2011-01-01', TRUE)VALUES[*]('Geschaeftsraum gross', 'tenant2', 1, 40.0, 400.0, '2011-01-01', TRUE)VALUES('Seitenfluegel unten', 'tenant3', 1, 40.0, 400.0, '2011-01-01', TRUE)VALUES('Seitenfluegel oben', 'tenant4', 1, 30.0, 300.0, '2011-01-01', TRUE)VALUES('Wohnung OG1', 'tenant5', 1, 120.0, 1000.0, '2011-01-01', TRUE)VALUES('Wohnung OG2 klein', 'tenant6', 1, 30.0, 370.0, '2011-01-01', TRUE)VALUES('Wohnung OG2 gross', 'tenant7', 1, 60.0, 700.0, '2011-01-01', TRUE)"
Syntax error in SQL statement "INSERT INTO FLAT(DESCRIPTION, TENANT, NUMRESIDENTS, SPACE, RENT, MOVINGDATE, ISRENTED) VALUES('Geschaeftsraum klein', 'tenant1', 1, 15.0, 200.0, '2011-01-01', TRUE)VALUES[*]('Geschaeftsraum gross', 'tenant2', 1, 40.0, 400.0, '2011-01-01', TRUE)VALUES('Seitenfluegel unten', 'tenant3', 1, 40.0, 400.0, '2011-01-01', TRUE)VALUES('Seitenfluegel oben', 'tenant4', 1, 30.0, 300.0, '2011-01-01', TRUE)VALUES('Wohnung OG1', 'tenant5', 1, 120.0, 1000.0, '2011-01-01', TRUE)VALUES('Wohnung OG2 klein', 'tenant6', 1, 30.0, 370.0, '2011-01-01', TRUE)VALUES('Wohnung OG2 gross', 'tenant7', 1, 60.0, 700.0, '2011-01-01', TRUE)"; SQL statement:
INSERT INTO Flat(description, tenant, numResidents, space, rent, movingDate, isRented) values('Geschaeftsraum klein', 'tenant1', 1, 15.0, 200.0, '2011-01-01', TRUE)values('Geschaeftsraum gross', 'tenant2', 1, 40.0, 400.0, '2011-01-01', TRUE)values('Seitenfluegel unten', 'tenant3', 1, 40.0, 400.0, '2011-01-01', TRUE)values('Seitenfluegel oben', 'tenant4', 1, 30.0, 300.0, '2011-01-01', TRUE)values('Wohnung OG1', 'tenant5', 1, 120.0, 1000.0, '2011-01-01', TRUE)values('Wohnung OG2 klein', 'tenant6', 1, 30.0, 370.0, '2011-01-01', TRUE)values('Wohnung OG2 gross', 'tenant7', 1, 60.0, 700.0, '2011-01-01', TRUE) [42000-200]
2019-11-15 21:41:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Anzahl der Felder stimmt nicht überein
Column count does not match; SQL statement:
INSERT INTO Flat(description, tenant, numResidents, space, rent, movingDate, isRented) values('Geschaeftsraum klein', 'tenant1', 1, 15.0, 200.0, '2011-01-01', TRUE), values('Geschaeftsraum gross', 'tenant2', 1, 40.0, 400.0, '2011-01-01', TRUE), values('Seitenfluegel unten', 'tenant3', 1, 40.0, 400.0, '2011-01-01', TRUE), values('Seitenfluegel oben', 'tenant4', 1, 30.0, 300.0, '2011-01-01', TRUE), values('Wohnung OG1', 'tenant5', 1, 120.0, 1000.0, '2011-01-01', TRUE), values('Wohnung OG2 klein', 'tenant6', 1, 30.0, 370.0, '2011-01-01', TRUE), values('Wohnung OG2 gross', 'tenant7', 1, 60.0, 700.0, '2011-01-01', TRUE) [21002-200]
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:205)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:181)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:170)
	at com.h2database@1.4.200/org.h2.command.Parser.parseValues(Parser.java:6464)
	at com.h2database@1.4.200/org.h2.command.Parser.parseQuerySub(Parser.java:2822)
	at com.h2database@1.4.200/org.h2.command.Parser.parseSelectUnion(Parser.java:2649)
	at com.h2database@1.4.200/org.h2.command.Parser.parseQuery(Parser.java:2620)
	at com.h2database@1.4.200/org.h2.command.Parser.readTerm(Parser.java:4431)
	at com.h2database@1.4.200/org.h2.command.Parser.readFactor(Parser.java:3343)
	at com.h2database@1.4.200/org.h2.command.Parser.readSum(Parser.java:3330)
	at com.h2database@1.4.200/org.h2.command.Parser.readConcat(Parser.java:3305)
	at com.h2database@1.4.200/org.h2.command.Parser.readCondition(Parser.java:3108)
	at com.h2database@1.4.200/org.h2.command.Parser.readExpression(Parser.java:3059)
	at com.h2database@1.4.200/org.h2.command.Parser.parseValues(Parser.java:6437)
	at com.h2database@1.4.200/org.h2.command.Parser.parseQuerySub(Parser.java:2822)
	at com.h2database@1.4.200/org.h2.command.Parser.parseSelectUnion(Parser.java:2649)
	at com.h2database@1.4.200/org.h2.command.Parser.parseQuery(Parser.java:2620)
	at com.h2database@1.4.200/org.h2.command.Parser.readTerm(Parser.java:4431)
	at com.h2database@1.4.200/org.h2.command.Parser.readFactor(Parser.java:3343)
	at com.h2database@1.4.200/org.h2.command.Parser.readSum(Parser.java:3330)
	at com.h2database@1.4.200/org.h2.command.Parser.readConcat(Parser.java:3305)
	at com.h2database@1.4.200/org.h2.command.Parser.readCondition(Parser.java:3108)
	at com.h2database@1.4.200/org.h2.command.Parser.readExpression(Parser.java:3059)
	at com.h2database@1.4.200/org.h2.command.Parser.parseValues(Parser.java:6437)
	at com.h2database@1.4.200/org.h2.command.Parser.parseQuerySub(Parser.java:2822)
	at com.h2database@1.4.200/org.h2.command.Parser.parseSelectUnion(Parser.java:2649)
	at com.h2database@1.4.200/org.h2.command.Parser.parseQuery(Parser.java:2620)
	at com.h2database@1.4.200/org.h2.command.Parser.readTerm(Parser.java:4431)
	at com.h2database@1.4.200/org.h2.command.Parser.readFactor(Parser.java:3343)
	at com.h2database@1.4.200/org.h2.command.Parser.readSum(Parser.java:3330)
	at com.h2database@1.4.200/org.h2.command.Parser.readConcat(Parser.java:3305)
	at com.h2database@1.4.200/org.h2.command.Parser.readCondition(Parser.java:3108)
	at com.h2database@1.4.200/org.h2.command.Parser.readExpression(Parser.java:3059)
	at com.h2database@1.4.200/org.h2.command.Parser.parseValues(Parser.java:6437)
	at com.h2database@1.4.200/org.h2.command.Parser.parseQuerySub(Parser.java:2822)
	at com.h2database@1.4.200/org.h2.command.Parser.parseSelectUnion(Parser.java:2649)
	at com.h2database@1.4.200/org.h2.command.Parser.parseQuery(Parser.java:2620)
	at com.h2database@1.4.200/org.h2.command.Parser.readTerm(Parser.java:4431)
	at com.h2database@1.4.200/org.h2.command.Parser.readFactor(Parser.java:3343)
	at com.h2database@1.4.200/org.h2.command.Parser.readSum(Parser.java:3330)
	at com.h2database@1.4.200/org.h2.command.Parser.readConcat(Parser.java:3305)
	at com.h2database@1.4.200/org.h2.command.Parser.readCondition(Parser.java:3108)
	at com.h2database@1.4.200/org.h2.command.Parser.readExpression(Parser.java:3059)
	at com.h2database@1.4.200/org.h2.command.Parser.parseValues(Parser.java:6437)
	at com.h2database@1.4.200/org.h2.command.Parser.parseQuerySub(Parser.java:2822)
	at com.h2database@1.4.200/org.h2.command.Parser.parseSelectUnion(Parser.java:2649)
	at com.h2database@1.4.200/org.h2.command.Parser.parseQuery(Parser.java:2620)
	at com.h2database@1.4.200/org.h2.command.Parser.readTerm(Parser.java:4431)
	at com.h2database@1.4.200/org.h2.command.Parser.readFactor(Parser.java:3343)
	at com.h2database@1.4.200/org.h2.command.Parser.readSum(Parser.java:3330)
	at com.h2database@1.4.200/org.h2.command.Parser.readConcat(Parser.java:3305)
	at com.h2database@1.4.200/org.h2.command.Parser.readCondition(Parser.java:3108)
	at com.h2database@1.4.200/org.h2.command.Parser.readExpression(Parser.java:3059)
	at com.h2database@1.4.200/org.h2.command.Parser.parseValuesForCommand(Parser.java:1881)
	at com.h2database@1.4.200/org.h2.command.Parser.parseInsertGivenTable(Parser.java:1817)
	at com.h2database@1.4.200/org.h2.command.Parser.parseInsert(Parser.java:1749)
	at com.h2database@1.4.200/org.h2.command.Parser.parsePrepared(Parser.java:954)
	at com.h2database@1.4.200/org.h2.command.Parser.parse(Parser.java:843)
	at com.h2database@1.4.200/org.h2.command.Parser.parse(Parser.java:815)
	at com.h2database@1.4.200/org.h2.command.Parser.prepareCommand(Parser.java:738)
	at com.h2database@1.4.200/org.h2.engine.Session.prepareLocal(Session.java:657)
	at com.h2database@1.4.200/org.h2.engine.Session.prepareCommand(Session.java:595)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:164)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:126)
	at com.utilities_statement/db.Insert.main(Insert.java:31)
2019-11-15 21:52:07 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "CREATE TABLE IF NOT EXISTS WATERMETER (ID INTEGER AUTO_INCREMENT, FLAT INT, DATE DATE, COLD FLOAT, WARM FLOAT, [*]"; erwartet "identifier"
Syntax error in SQL statement "CREATE TABLE IF NOT EXISTS WATERMETER (ID INTEGER AUTO_INCREMENT, FLAT INT, DATE DATE, COLD FLOAT, WARM FLOAT, [*]"; expected "identifier"; SQL statement:
CREATE TABLE IF NOT EXISTS watermeter (id INTEGER AUTO_INCREMENT, flat INT, date DATE, cold FLOAT, warm FLOAT,  [42001-200]
2019-11-15 21:59:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "INSERT INTO WATERMETER(FLATID, DATE, COLD, WARM) VALUES(0, '2018-01-01', 23.0, 75.0), (1, '2018-01-01', 55.0, 102.0), (0, '2019-01-01', 34.0, 98.0), (1, '2019-01-01', 61.0, 115.0), [*]"; erwartet "ROW, (, DEFAULT, NOT, EXISTS, INTERSECTS, UNIQUE"
Syntax error in SQL statement "INSERT INTO WATERMETER(FLATID, DATE, COLD, WARM) VALUES(0, '2018-01-01', 23.0, 75.0), (1, '2018-01-01', 55.0, 102.0), (0, '2019-01-01', 34.0, 98.0), (1, '2019-01-01', 61.0, 115.0), [*]"; expected "ROW, (, DEFAULT, NOT, EXISTS, INTERSECTS, UNIQUE"; SQL statement:
INSERT INTO WaterMeter(flatID, date, cold, warm) values(0, '2018-01-01', 23.0, 75.0), (1, '2018-01-01', 55.0, 102.0), (0, '2019-01-01', 34.0, 98.0), (1, '2019-01-01', 61.0, 115.0),  [42001-200]
2019-11-15 22:06:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Unerlaubter Wert "6" für Parameter "columnIndex"
Invalid value "6" for parameter "columnIndex" [90008-200]
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:590)
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:205)
	at com.h2database@1.4.200/org.h2.message.DbException.getInvalidValueException(DbException.java:280)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcResultSet.checkColumnIndex(JdbcResultSet.java:3212)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3251)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcResultSet.getFloat(JdbcResultSet.java:698)
	at com.utilities_statement/db.ShowContentOfTableWaterMeter.main(ShowContentOfTableWaterMeter.java:27)
2019-11-15 22:14:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "INSERT INTO WATERMETER(FLATID, DATE, COLD, WARM) VALUES((SELECT ID FROM FLAT WHERE ID = 0), '2018-01-01', 23.0, 75.0), ((SELECT ID FROM FLAT WHERE ID = 1), '2018-01-01', 55.0, 102.0), ((SELECT ID FROM FLAT WHERE ID = 0), '2019-01-01', 34.0, 98.0), ((SELECT ID FROM FLAT WHERE ID = 1), '2019-01-01', 61.0, 115.0) )[*]"
Syntax error in SQL statement "INSERT INTO WATERMETER(FLATID, DATE, COLD, WARM) VALUES((SELECT ID FROM FLAT WHERE ID = 0), '2018-01-01', 23.0, 75.0), ((SELECT ID FROM FLAT WHERE ID = 1), '2018-01-01', 55.0, 102.0), ((SELECT ID FROM FLAT WHERE ID = 0), '2019-01-01', 34.0, 98.0), ((SELECT ID FROM FLAT WHERE ID = 1), '2019-01-01', 61.0, 115.0) )[*]"; SQL statement:
INSERT INTO WaterMeter(flatID, date, cold, warm) values((SELECT id FROM flat WHERE id = 0), '2018-01-01', 23.0, 75.0), ((SELECT id FROM flat WHERE id = 1), '2018-01-01', 55.0, 102.0), ((SELECT id FROM flat WHERE id = 0), '2019-01-01', 34.0, 98.0), ((SELECT id FROM flat WHERE id = 1), '2019-01-01', 61.0, 115.0) ) [42000-200]
2019-11-16 15:54:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "INSERT INTO billitem(description, value, unit, year) values('Strassenreinigung', 12.54, 'FLAT', 2018), ('Abfallentsorgung', 1200.0, 'NUMRESIDENTS', 2018), ('Treppenhausreinigung', 227.33, 'NUMRESIDENTS', 2018), ('Treppenhausreinigung', '230.0, 'NUMRESIDENTS[*]', 2019)"
Syntax error in SQL statement "INSERT INTO billitem(description, value, unit, year) values('Strassenreinigung', 12.54, 'FLAT', 2018), ('Abfallentsorgung', 1200.0, 'NUMRESIDENTS', 2018), ('Treppenhausreinigung', 227.33, 'NUMRESIDENTS', 2018), ('Treppenhausreinigung', '230.0, 'NUMRESIDENTS[*]', 2019)"; SQL statement:
INSERT INTO billitem(description, value, unit, year) values('Strassenreinigung', 12.54, 'FLAT', 2018), ('Abfallentsorgung', 1200.0, 'NUMRESIDENTS', 2018), ('Treppenhausreinigung', 227.33, 'NUMRESIDENTS', 2018), ('Treppenhausreinigung', '230.0, 'NUMRESIDENTS', 2019) [42000-200]
2019-11-16 15:57:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "INSERT INTO billitem(description, value, unit, year) values('Strassenreinigung', 12.54, 'FLAT', '2018-01-01'), ('Abfallentsorgung', 1200.0, 'NUMRESIDENTS', '2018-01-01'), ('Treppenhausreinigung', 227.33, 'NUMRESIDENTS', '2018-01-01'), ('Treppenhausreinigung', '230.0, 'NUMRESIDENTS', '2019-01-01[*]')"
Syntax error in SQL statement "INSERT INTO billitem(description, value, unit, year) values('Strassenreinigung', 12.54, 'FLAT', '2018-01-01'), ('Abfallentsorgung', 1200.0, 'NUMRESIDENTS', '2018-01-01'), ('Treppenhausreinigung', 227.33, 'NUMRESIDENTS', '2018-01-01'), ('Treppenhausreinigung', '230.0, 'NUMRESIDENTS', '2019-01-01[*]')"; SQL statement:
INSERT INTO billitem(description, value, unit, year) values('Strassenreinigung', 12.54, 'FLAT', '2018-01-01'), ('Abfallentsorgung', 1200.0, 'NUMRESIDENTS', '2018-01-01'), ('Treppenhausreinigung', 227.33, 'NUMRESIDENTS', '2018-01-01'), ('Treppenhausreinigung', '230.0, 'NUMRESIDENTS', '2019-01-01') [42000-200]
2019-11-16 16:04:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabelle "BILLITEM" nicht gefunden
Table "BILLITEM" not found; SQL statement:
INSERT INTO billitem(description, value, unit, year) values('Strassenreinigung', 12.54, 'FLAT', '2018-01-01') [42102-200]
2019-11-16 16:04:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabelle "BILLITEM" nicht gefunden
Table "BILLITEM" not found; SQL statement:
INSERT INTO billitem(description, value, unit, year) values('Strassenreinigung', 12.54, 'FLAT', '2018-01-01') [42102-200]
2019-11-16 16:05:48 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "INSERT INTO billitem(description, value, unit, year) values('Strassenreinigung', 12.54, 'FLAT', '2018-01-01'), ('Abfallentsorgung', 1200.0, 'NUMRESIDENTS', '2018-01-01'), ('Treppenhausreinigung', 227.33, 'NUMRESIDENTS', '2018-01-01'), ('Treppenhausreinigung', '230.0, 'NUMRESIDENTS', '2019-01-01[*]')"
Syntax error in SQL statement "INSERT INTO billitem(description, value, unit, year) values('Strassenreinigung', 12.54, 'FLAT', '2018-01-01'), ('Abfallentsorgung', 1200.0, 'NUMRESIDENTS', '2018-01-01'), ('Treppenhausreinigung', 227.33, 'NUMRESIDENTS', '2018-01-01'), ('Treppenhausreinigung', '230.0, 'NUMRESIDENTS', '2019-01-01[*]')"; SQL statement:
INSERT INTO billitem(description, value, unit, year) values('Strassenreinigung', 12.54, 'FLAT', '2018-01-01'), ('Abfallentsorgung', 1200.0, 'NUMRESIDENTS', '2018-01-01'), ('Treppenhausreinigung', 227.33, 'NUMRESIDENTS', '2018-01-01'), ('Treppenhausreinigung', '230.0, 'NUMRESIDENTS', '2019-01-01') [42000-200]
2019-11-16 16:06:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "INSERT INTO billitem(description, value, unit, year) values('Strassenreinigung', 12.54, 'FLAT', '2018-01-01'), ('Abfallentsorgung', 1200.0, 'NUMRESIDENTS', '2018-01-01'), ('Treppenhausreinigung', 227.33, 'NUMRESIDENTS', '2018-01-01'), ('Treppenhausreinigung', '230.0, 'NUMRESIDENTS', '2019-01-01[*]')"
Syntax error in SQL statement "INSERT INTO billitem(description, value, unit, year) values('Strassenreinigung', 12.54, 'FLAT', '2018-01-01'), ('Abfallentsorgung', 1200.0, 'NUMRESIDENTS', '2018-01-01'), ('Treppenhausreinigung', 227.33, 'NUMRESIDENTS', '2018-01-01'), ('Treppenhausreinigung', '230.0, 'NUMRESIDENTS', '2019-01-01[*]')"; SQL statement:
INSERT INTO billitem(description, value, unit, year) values('Strassenreinigung', 12.54, 'FLAT', '2018-01-01'), ('Abfallentsorgung', 1200.0, 'NUMRESIDENTS', '2018-01-01'), ('Treppenhausreinigung', 227.33, 'NUMRESIDENTS', '2018-01-01'), ('Treppenhausreinigung', '230.0, 'NUMRESIDENTS', '2019-01-01') [42000-200]
2019-11-16 16:28:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Feld "ARTICLE" nicht gefunden
Column "ARTICLE" not found; SQL statement:
INSERT INTO billitem(article, value, unit, year) values('GRUNDSTEUER', 1222.54, 'FLAT', '2018-01-01'), ('GEBAEUDEVERSICHERUNG', 1200.0, 'NUMRESIDENTS', '2018-01-01'), ('STRASSENREINIGUNG', 227.33, 'NUMRESIDENTS', '2018-01-01'), ('STRASSENREINIGUNG', 230.0, 'NUMRESIDENTS', '2019-01-01') [42122-200]
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:205)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:181)
	at com.h2database@1.4.200/org.h2.table.Table.getColumn(Table.java:703)
	at com.h2database@1.4.200/org.h2.command.Parser.parseColumn(Parser.java:1450)
	at com.h2database@1.4.200/org.h2.command.Parser.parseColumnList(Parser.java:1435)
	at com.h2database@1.4.200/org.h2.command.Parser.parseInsertGivenTable(Parser.java:1804)
	at com.h2database@1.4.200/org.h2.command.Parser.parseInsert(Parser.java:1749)
	at com.h2database@1.4.200/org.h2.command.Parser.parsePrepared(Parser.java:954)
	at com.h2database@1.4.200/org.h2.command.Parser.parse(Parser.java:843)
	at com.h2database@1.4.200/org.h2.command.Parser.parse(Parser.java:815)
	at com.h2database@1.4.200/org.h2.command.Parser.prepareCommand(Parser.java:738)
	at com.h2database@1.4.200/org.h2.engine.Session.prepareLocal(Session.java:657)
	at com.h2database@1.4.200/org.h2.engine.Session.prepareCommand(Session.java:595)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:164)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:126)
	at com.utilities_statement/dbinit.InsertIntoBillItem.main(InsertIntoBillItem.java:26)
2019-11-16 17:11:09 jdbc[11]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "SELECT * FROM BILLITEM WHERE YEAR = THU FEB[*] 01 00:00:00 CET 2018"
Syntax error in SQL statement "SELECT * FROM BILLITEM WHERE YEAR = THU FEB[*] 01 00:00:00 CET 2018"; SQL statement:
SELECT * FROM billitem WHERE year = Thu Feb 01 00:00:00 CET 2018 [42000-200]
2019-11-16 17:16:11 jdbc[11]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "SELECT * FROM BILLITEM WHERE YEAR = THU FEB[*] 01 00:00:00 CET 2018"
Syntax error in SQL statement "SELECT * FROM BILLITEM WHERE YEAR = THU FEB[*] 01 00:00:00 CET 2018"; SQL statement:
SELECT * FROM billitem WHERE year = Thu Feb 01 00:00:00 CET 2018 [42000-200]
2019-11-16 17:19:39 jdbc[11]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "SELECT * FROM BILLITEM WHERE YEAR = THU FEB[*] 01 00:00:00 CET 2018"
Syntax error in SQL statement "SELECT * FROM BILLITEM WHERE YEAR = THU FEB[*] 01 00:00:00 CET 2018"; SQL statement:
SELECT * FROM billitem WHERE year = Thu Feb 01 00:00:00 CET 2018 [42000-200]
2019-11-16 17:23:03 jdbc[11]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "SELECT * FROM BILLITEM WHERE YEAR = THU FEB[*] 01 00:00:00 CET 2018"
Syntax error in SQL statement "SELECT * FROM BILLITEM WHERE YEAR = THU FEB[*] 01 00:00:00 CET 2018"; SQL statement:
SELECT * FROM billitem WHERE year = Thu Feb 01 00:00:00 CET 2018 [42000-200]
2019-11-17 09:35:26 jdbc[11]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "SELECT * FROM BILLITEM WHERE YEAR = THU FEB[*] 01 00:00:00 CET 2018"
Syntax error in SQL statement "SELECT * FROM BILLITEM WHERE YEAR = THU FEB[*] 01 00:00:00 CET 2018"; SQL statement:
SELECT * FROM billitem WHERE year = Thu Feb 01 00:00:00 CET 2018 [42000-200]
2019-11-17 09:36:43 jdbc[11]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "SELECT * FROM BILLITEM WHERE YEAR = THU FEB[*] 01 00:00:00 CET 2018"
Syntax error in SQL statement "SELECT * FROM BILLITEM WHERE YEAR = THU FEB[*] 01 00:00:00 CET 2018"; SQL statement:
SELECT * FROM billitem WHERE year = Thu Feb 01 00:00:00 CET 2018 [42000-200]
2019-11-17 09:42:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Kann "PUBLIC.FLAT" nicht zurücksetzen per TRUNCATE
Cannot truncate "PUBLIC.FLAT"; SQL statement:
TRUNCATE TABLE Flat [90106-200]
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:205)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:181)
	at com.h2database@1.4.200/org.h2.command.ddl.TruncateTable.update(TruncateTable.java:43)
	at com.h2database@1.4.200/org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at com.h2database@1.4.200/org.h2.command.Command.executeUpdate(Command.java:251)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:168)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:126)
	at com.utilities_statement/dbinit.ClearTableFlat.main(ClearTableFlat.java:22)
2019-11-17 09:44:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Kann "FLAT" nicht löschen weil "CONSTRAINT_92" davon abhängt
Cannot drop "FLAT" because "CONSTRAINT_92" depends on it; SQL statement:
DROP TABLE IF EXISTS flat [90107-200]
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:205)
	at com.h2database@1.4.200/org.h2.command.ddl.DropTable.prepareDrop(DropTable.java:98)
	at com.h2database@1.4.200/org.h2.command.ddl.DropTable.update(DropTable.java:124)
	at com.h2database@1.4.200/org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at com.h2database@1.4.200/org.h2.command.Command.executeUpdate(Command.java:251)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:168)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:126)
	at com.utilities_statement/dbinit.DeleteTableFlat.main(DeleteTableFlat.java:22)
2019-11-17 09:44:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabelle "WATERMETER" nicht gefunden
Table "WATERMETER" not found; SQL statement:
INSERT INTO WaterMeter(flatID, date, cold, warm, addcon) values((SELECT id FROM flat WHERE id = 0), '2018-01-01', 23.0, 75.0, 5.0), ((SELECT id FROM flat WHERE id = 1), '2018-01-01', 55.0, 102.0, 5.0), ((SELECT id FROM flat WHERE id = 0), '2019-01-01', 34.0, 98.0, 5.0), ((SELECT id FROM flat WHERE id = 1), '2019-01-01', 61.0, 115.0, 5.0) [42102-200]
2019-11-17 09:44:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabelle "FLAT" nicht gefunden
Table "FLAT" not found; SQL statement:
TRUNCATE TABLE Flat [42102-200]
2019-11-17 09:47:29 jdbc[4]: exception
org.h2.jdbc.JdbcSQLDataException: Kann "DATE" "1" nicht umwandeln
Cannot parse "DATE" constant "1" [22007-200]
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:194)
	at com.h2database@1.4.200/org.h2.value.ValueDate.parse(ValueDate.java:74)
	at com.h2database@1.4.200/org.h2.value.Value.convertToDate(Value.java:1093)
	at com.h2database@1.4.200/org.h2.value.Value.convertTo(Value.java:816)
	at com.h2database@1.4.200/org.h2.value.Value.convertTo(Value.java:737)
	at com.h2database@1.4.200/org.h2.value.Value.getDate(Value.java:571)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:388)
	at com.utilities_statement/com.utilities_statement.persistent.LoadWaterMeters.loadWaterMeters(LoadWaterMeters.java:26)
	at com.utilities_statement/com.utilities_statement.persistent.LoadFlats.loadFlats(LoadFlats.java:36)
	at com.utilities_statement/com.utilities_statement.client.Main.main(Main.java:27)
Caused by: java.lang.IllegalArgumentException: 1
	at com.h2database@1.4.200/org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:166)
	at com.h2database@1.4.200/org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 8 more
2019-11-17 09:47:29 jdbc[11]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax Fehler in SQL Befehl "SELECT * FROM BILLITEM WHERE YEAR = THU FEB[*] 01 00:00:00 CET 2018"
Syntax error in SQL statement "SELECT * FROM BILLITEM WHERE YEAR = THU FEB[*] 01 00:00:00 CET 2018"; SQL statement:
SELECT * FROM billitem WHERE year = Thu Feb 01 00:00:00 CET 2018 [42000-200]
2019-11-17 09:51:07 jdbc[4]: exception
org.h2.jdbc.JdbcSQLDataException: Kann "DATE" "1" nicht umwandeln
Cannot parse "DATE" constant "1" [22007-200]
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:194)
	at com.h2database@1.4.200/org.h2.value.ValueDate.parse(ValueDate.java:74)
	at com.h2database@1.4.200/org.h2.value.Value.convertToDate(Value.java:1093)
	at com.h2database@1.4.200/org.h2.value.Value.convertTo(Value.java:816)
	at com.h2database@1.4.200/org.h2.value.Value.convertTo(Value.java:737)
	at com.h2database@1.4.200/org.h2.value.Value.getDate(Value.java:571)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:388)
	at com.utilities_statement/com.utilities_statement.persistent.LoadWaterMeters.loadWaterMeters(LoadWaterMeters.java:26)
	at com.utilities_statement/com.utilities_statement.persistent.LoadFlats.loadFlats(LoadFlats.java:36)
	at com.utilities_statement/com.utilities_statement.client.Main.main(Main.java:26)
Caused by: java.lang.IllegalArgumentException: 1
	at com.h2database@1.4.200/org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:166)
	at com.h2database@1.4.200/org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 8 more
2019-11-17 09:51:07 jdbc[11]: exception
org.h2.jdbc.JdbcSQLDataException: Kann "DATE" "2016" nicht umwandeln
Cannot parse "DATE" constant "2016"; SQL statement:
SELECT * FROM billitem WHERE year = 2018-01-01 [22007-200]
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:194)
	at com.h2database@1.4.200/org.h2.value.ValueDate.parse(ValueDate.java:74)
	at com.h2database@1.4.200/org.h2.value.Value.convertToDate(Value.java:1093)
	at com.h2database@1.4.200/org.h2.value.Value.convertTo(Value.java:816)
	at com.h2database@1.4.200/org.h2.value.Value.convertTo(Value.java:772)
	at com.h2database@1.4.200/org.h2.expression.condition.Comparison.optimize(Comparison.java:221)
	at com.h2database@1.4.200/org.h2.command.dml.Select.prepare(Select.java:1209)
	at com.h2database@1.4.200/org.h2.command.Parser.prepareCommand(Parser.java:744)
	at com.h2database@1.4.200/org.h2.engine.Session.prepareLocal(Session.java:657)
	at com.h2database@1.4.200/org.h2.engine.Session.prepareCommand(Session.java:595)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcStatement.executeQuery(JdbcStatement.java:78)
	at com.utilities_statement/com.utilities_statement.persistent.DatabaseController.runQuery(DatabaseController.java:24)
	at com.utilities_statement/com.utilities_statement.persistent.LoadBillItems.loadBillItems(LoadBillItems.java:26)
	at com.utilities_statement/com.utilities_statement.client.Main.main(Main.java:28)
Caused by: java.lang.IllegalArgumentException: 2016
	at com.h2database@1.4.200/org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:166)
	at com.h2database@1.4.200/org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 13 more
2019-11-17 09:51:59 jdbc[4]: exception
org.h2.jdbc.JdbcSQLDataException: Kann "DATE" "1" nicht umwandeln
Cannot parse "DATE" constant "1" [22007-200]
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:194)
	at com.h2database@1.4.200/org.h2.value.ValueDate.parse(ValueDate.java:74)
	at com.h2database@1.4.200/org.h2.value.Value.convertToDate(Value.java:1093)
	at com.h2database@1.4.200/org.h2.value.Value.convertTo(Value.java:816)
	at com.h2database@1.4.200/org.h2.value.Value.convertTo(Value.java:737)
	at com.h2database@1.4.200/org.h2.value.Value.getDate(Value.java:571)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:388)
	at com.utilities_statement/com.utilities_statement.persistent.LoadWaterMeters.loadWaterMeters(LoadWaterMeters.java:26)
	at com.utilities_statement/com.utilities_statement.persistent.LoadFlats.loadFlats(LoadFlats.java:36)
	at com.utilities_statement/com.utilities_statement.client.Main.main(Main.java:26)
Caused by: java.lang.IllegalArgumentException: 1
	at com.h2database@1.4.200/org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:166)
	at com.h2database@1.4.200/org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 8 more
2019-11-17 09:51:59 jdbc[11]: exception
org.h2.jdbc.JdbcSQLDataException: Kann "DATE" "2016" nicht umwandeln
Cannot parse "DATE" constant "2016"; SQL statement:
SELECT * FROM billitem WHERE year = 2018-01-01 [22007-200]
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:194)
	at com.h2database@1.4.200/org.h2.value.ValueDate.parse(ValueDate.java:74)
	at com.h2database@1.4.200/org.h2.value.Value.convertToDate(Value.java:1093)
	at com.h2database@1.4.200/org.h2.value.Value.convertTo(Value.java:816)
	at com.h2database@1.4.200/org.h2.value.Value.convertTo(Value.java:772)
	at com.h2database@1.4.200/org.h2.expression.condition.Comparison.optimize(Comparison.java:221)
	at com.h2database@1.4.200/org.h2.command.dml.Select.prepare(Select.java:1209)
	at com.h2database@1.4.200/org.h2.command.Parser.prepareCommand(Parser.java:744)
	at com.h2database@1.4.200/org.h2.engine.Session.prepareLocal(Session.java:657)
	at com.h2database@1.4.200/org.h2.engine.Session.prepareCommand(Session.java:595)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcStatement.executeQuery(JdbcStatement.java:78)
	at com.utilities_statement/com.utilities_statement.persistent.DatabaseController.runQuery(DatabaseController.java:24)
	at com.utilities_statement/com.utilities_statement.persistent.LoadBillItems.loadBillItems(LoadBillItems.java:26)
	at com.utilities_statement/com.utilities_statement.client.Main.main(Main.java:28)
Caused by: java.lang.IllegalArgumentException: 2016
	at com.h2database@1.4.200/org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:166)
	at com.h2database@1.4.200/org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 13 more
2019-11-17 09:58:35 jdbc[4]: exception
org.h2.jdbc.JdbcSQLDataException: Kann "DATE" "1" nicht umwandeln
Cannot parse "DATE" constant "1" [22007-200]
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:194)
	at com.h2database@1.4.200/org.h2.value.ValueDate.parse(ValueDate.java:74)
	at com.h2database@1.4.200/org.h2.value.Value.convertToDate(Value.java:1093)
	at com.h2database@1.4.200/org.h2.value.Value.convertTo(Value.java:816)
	at com.h2database@1.4.200/org.h2.value.Value.convertTo(Value.java:737)
	at com.h2database@1.4.200/org.h2.value.Value.getDate(Value.java:571)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:388)
	at com.utilities_statement/com.utilities_statement.persistent.LoadWaterMeters.loadWaterMeters(LoadWaterMeters.java:26)
	at com.utilities_statement/com.utilities_statement.persistent.LoadFlats.loadFlats(LoadFlats.java:36)
	at com.utilities_statement/com.utilities_statement.client.Main.main(Main.java:25)
Caused by: java.lang.IllegalArgumentException: 1
	at com.h2database@1.4.200/org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:166)
	at com.h2database@1.4.200/org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 8 more
2019-11-17 09:58:35 jdbc[11]: exception
org.h2.jdbc.JdbcSQLDataException: Kann "DATE" "2016" nicht umwandeln
Cannot parse "DATE" constant "2016"; SQL statement:
SELECT * FROM billitem WHERE year = 2018-01-01 [22007-200]
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:194)
	at com.h2database@1.4.200/org.h2.value.ValueDate.parse(ValueDate.java:74)
	at com.h2database@1.4.200/org.h2.value.Value.convertToDate(Value.java:1093)
	at com.h2database@1.4.200/org.h2.value.Value.convertTo(Value.java:816)
	at com.h2database@1.4.200/org.h2.value.Value.convertTo(Value.java:772)
	at com.h2database@1.4.200/org.h2.expression.condition.Comparison.optimize(Comparison.java:221)
	at com.h2database@1.4.200/org.h2.command.dml.Select.prepare(Select.java:1209)
	at com.h2database@1.4.200/org.h2.command.Parser.prepareCommand(Parser.java:744)
	at com.h2database@1.4.200/org.h2.engine.Session.prepareLocal(Session.java:657)
	at com.h2database@1.4.200/org.h2.engine.Session.prepareCommand(Session.java:595)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcStatement.executeQuery(JdbcStatement.java:78)
	at com.utilities_statement/com.utilities_statement.persistent.DatabaseController.runQuery(DatabaseController.java:24)
	at com.utilities_statement/com.utilities_statement.persistent.LoadBillItems.loadBillItems(LoadBillItems.java:26)
	at com.utilities_statement/com.utilities_statement.client.Main.main(Main.java:27)
Caused by: java.lang.IllegalArgumentException: 2016
	at com.h2database@1.4.200/org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:166)
	at com.h2database@1.4.200/org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 13 more
2019-11-17 10:01:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Kann "DATE" "2017" nicht umwandeln
Cannot parse "DATE" constant "2017"; SQL statement:
SELECT * FROM billitem WHERE year = 2019-01-01 [22007-200]
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:194)
	at com.h2database@1.4.200/org.h2.value.ValueDate.parse(ValueDate.java:74)
	at com.h2database@1.4.200/org.h2.value.Value.convertToDate(Value.java:1093)
	at com.h2database@1.4.200/org.h2.value.Value.convertTo(Value.java:816)
	at com.h2database@1.4.200/org.h2.value.Value.convertTo(Value.java:772)
	at com.h2database@1.4.200/org.h2.expression.condition.Comparison.optimize(Comparison.java:221)
	at com.h2database@1.4.200/org.h2.command.dml.Select.prepare(Select.java:1209)
	at com.h2database@1.4.200/org.h2.command.Parser.prepareCommand(Parser.java:744)
	at com.h2database@1.4.200/org.h2.engine.Session.prepareLocal(Session.java:657)
	at com.h2database@1.4.200/org.h2.engine.Session.prepareCommand(Session.java:595)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcStatement.executeQuery(JdbcStatement.java:78)
	at com.utilities_statement/com.utilities_statement.persistent.DatabaseController.runQuery(DatabaseController.java:24)
	at com.utilities_statement/com.utilities_statement.persistent.LoadBillItems.loadBillItems(LoadBillItems.java:26)
	at com.utilities_statement/com.utilities_statement.LoadBillItemsTest.testLoadBillItems(LoadBillItemsTest.java:33)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at junit.framework.TestCase.runTest(TestCase.java:154)
	at junit.framework.TestCase.runBare(TestCase.java:127)
	at junit.framework.TestResult$1.protect(TestResult.java:106)
	at junit.framework.TestResult.runProtected(TestResult.java:124)
	at junit.framework.TestResult.run(TestResult.java:109)
	at junit.framework.TestCase.run(TestCase.java:118)
	at junit.framework.TestSuite.runTest(TestSuite.java:208)
	at junit.framework.TestSuite.run(TestSuite.java:203)
	at org.eclipse.jdt.internal.junit.runner.junit3.JUnit3TestReference.run(JUnit3TestReference.java:124)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:541)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:763)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:463)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:209)
Caused by: java.lang.IllegalArgumentException: 2017
	at com.h2database@1.4.200/org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:166)
	at com.h2database@1.4.200/org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 31 more
2019-11-17 10:03:14 jdbc[4]: exception
org.h2.jdbc.JdbcSQLDataException: Kann "DATE" "1" nicht umwandeln
Cannot parse "DATE" constant "1" [22007-200]
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:194)
	at com.h2database@1.4.200/org.h2.value.ValueDate.parse(ValueDate.java:74)
	at com.h2database@1.4.200/org.h2.value.Value.convertToDate(Value.java:1093)
	at com.h2database@1.4.200/org.h2.value.Value.convertTo(Value.java:816)
	at com.h2database@1.4.200/org.h2.value.Value.convertTo(Value.java:737)
	at com.h2database@1.4.200/org.h2.value.Value.getDate(Value.java:571)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:388)
	at com.utilities_statement/com.utilities_statement.persistent.LoadWaterMeters.loadWaterMeters(LoadWaterMeters.java:26)
	at com.utilities_statement/com.utilities_statement.persistent.LoadFlats.loadFlats(LoadFlats.java:36)
	at com.utilities_statement/com.utilities_statement.client.Main.main(Main.java:25)
Caused by: java.lang.IllegalArgumentException: 1
	at com.h2database@1.4.200/org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:166)
	at com.h2database@1.4.200/org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 8 more
2019-11-17 10:03:39 jdbc[4]: exception
org.h2.jdbc.JdbcSQLDataException: Kann "DATE" "1" nicht umwandeln
Cannot parse "DATE" constant "1" [22007-200]
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:194)
	at com.h2database@1.4.200/org.h2.value.ValueDate.parse(ValueDate.java:74)
	at com.h2database@1.4.200/org.h2.value.Value.convertToDate(Value.java:1093)
	at com.h2database@1.4.200/org.h2.value.Value.convertTo(Value.java:816)
	at com.h2database@1.4.200/org.h2.value.Value.convertTo(Value.java:737)
	at com.h2database@1.4.200/org.h2.value.Value.getDate(Value.java:571)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:388)
	at com.utilities_statement/com.utilities_statement.persistent.LoadWaterMeters.loadWaterMeters(LoadWaterMeters.java:26)
	at com.utilities_statement/com.utilities_statement.persistent.LoadFlats.loadFlats(LoadFlats.java:36)
	at com.utilities_statement/com.utilities_statement.client.Main.main(Main.java:25)
Caused by: java.lang.IllegalArgumentException: 1
	at com.h2database@1.4.200/org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:166)
	at com.h2database@1.4.200/org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 8 more
2019-11-17 10:03:57 jdbc[4]: exception
org.h2.jdbc.JdbcSQLDataException: Kann "DATE" "1" nicht umwandeln
Cannot parse "DATE" constant "1" [22007-200]
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:194)
	at com.h2database@1.4.200/org.h2.value.ValueDate.parse(ValueDate.java:74)
	at com.h2database@1.4.200/org.h2.value.Value.convertToDate(Value.java:1093)
	at com.h2database@1.4.200/org.h2.value.Value.convertTo(Value.java:816)
	at com.h2database@1.4.200/org.h2.value.Value.convertTo(Value.java:737)
	at com.h2database@1.4.200/org.h2.value.Value.getDate(Value.java:571)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:388)
	at com.utilities_statement/com.utilities_statement.persistent.LoadWaterMeters.loadWaterMeters(LoadWaterMeters.java:26)
	at com.utilities_statement/com.utilities_statement.persistent.LoadFlats.loadFlats(LoadFlats.java:36)
	at com.utilities_statement/com.utilities_statement.client.Main.main(Main.java:25)
Caused by: java.lang.IllegalArgumentException: 1
	at com.h2database@1.4.200/org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:166)
	at com.h2database@1.4.200/org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 8 more
2019-11-17 10:09:11 jdbc[4]: exception
org.h2.jdbc.JdbcSQLDataException: Kann "DATE" "1" nicht umwandeln
Cannot parse "DATE" constant "1" [22007-200]
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:194)
	at com.h2database@1.4.200/org.h2.value.ValueDate.parse(ValueDate.java:74)
	at com.h2database@1.4.200/org.h2.value.Value.convertToDate(Value.java:1093)
	at com.h2database@1.4.200/org.h2.value.Value.convertTo(Value.java:816)
	at com.h2database@1.4.200/org.h2.value.Value.convertTo(Value.java:737)
	at com.h2database@1.4.200/org.h2.value.Value.getDate(Value.java:571)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:388)
	at com.utilities_statement/com.utilities_statement.persistent.LoadWaterMeters.loadWaterMeters(LoadWaterMeters.java:26)
	at com.utilities_statement/com.utilities_statement.persistent.LoadFlats.loadFlats(LoadFlats.java:36)
	at com.utilities_statement/com.utilities_statement.LoadFlatsTest.testLoadFlats(LoadFlatsTest.java:34)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at junit.framework.TestCase.runTest(TestCase.java:154)
	at junit.framework.TestCase.runBare(TestCase.java:127)
	at junit.framework.TestResult$1.protect(TestResult.java:106)
	at junit.framework.TestResult.runProtected(TestResult.java:124)
	at junit.framework.TestResult.run(TestResult.java:109)
	at junit.framework.TestCase.run(TestCase.java:118)
	at junit.framework.TestSuite.runTest(TestSuite.java:208)
	at junit.framework.TestSuite.run(TestSuite.java:203)
	at org.eclipse.jdt.internal.junit.runner.junit3.JUnit3TestReference.run(JUnit3TestReference.java:124)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:541)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:763)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:463)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:209)
Caused by: java.lang.IllegalArgumentException: 1
	at com.h2database@1.4.200/org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:166)
	at com.h2database@1.4.200/org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 26 more
2019-11-17 10:09:26 jdbc[4]: exception
org.h2.jdbc.JdbcSQLDataException: Kann "DATE" "1" nicht umwandeln
Cannot parse "DATE" constant "1" [22007-200]
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:194)
	at com.h2database@1.4.200/org.h2.value.ValueDate.parse(ValueDate.java:74)
	at com.h2database@1.4.200/org.h2.value.Value.convertToDate(Value.java:1093)
	at com.h2database@1.4.200/org.h2.value.Value.convertTo(Value.java:816)
	at com.h2database@1.4.200/org.h2.value.Value.convertTo(Value.java:737)
	at com.h2database@1.4.200/org.h2.value.Value.getDate(Value.java:571)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:388)
	at com.utilities_statement/com.utilities_statement.persistent.LoadWaterMeters.loadWaterMeters(LoadWaterMeters.java:26)
	at com.utilities_statement/com.utilities_statement.persistent.LoadFlats.loadFlats(LoadFlats.java:36)
	at com.utilities_statement/com.utilities_statement.LoadFlatsTest.testLoadFlats(LoadFlatsTest.java:34)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at junit.framework.TestCase.runTest(TestCase.java:154)
	at junit.framework.TestCase.runBare(TestCase.java:127)
	at junit.framework.TestResult$1.protect(TestResult.java:106)
	at junit.framework.TestResult.runProtected(TestResult.java:124)
	at junit.framework.TestResult.run(TestResult.java:109)
	at junit.framework.TestCase.run(TestCase.java:118)
	at junit.framework.TestSuite.runTest(TestSuite.java:208)
	at junit.framework.TestSuite.run(TestSuite.java:203)
	at org.eclipse.jdt.internal.junit.runner.junit3.JUnit3TestReference.run(JUnit3TestReference.java:124)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:541)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:763)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:463)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:209)
Caused by: java.lang.IllegalArgumentException: 1
	at com.h2database@1.4.200/org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:166)
	at com.h2database@1.4.200/org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 26 more
2019-11-17 10:12:46 jdbc[4]: exception
org.h2.jdbc.JdbcSQLDataException: Kann "DATE" "1" nicht umwandeln
Cannot parse "DATE" constant "1" [22007-200]
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:194)
	at com.h2database@1.4.200/org.h2.value.ValueDate.parse(ValueDate.java:74)
	at com.h2database@1.4.200/org.h2.value.Value.convertToDate(Value.java:1093)
	at com.h2database@1.4.200/org.h2.value.Value.convertTo(Value.java:816)
	at com.h2database@1.4.200/org.h2.value.Value.convertTo(Value.java:737)
	at com.h2database@1.4.200/org.h2.value.Value.getDate(Value.java:571)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:388)
	at com.utilities_statement/com.utilities_statement.persistent.LoadWaterMeters.loadWaterMeters(LoadWaterMeters.java:26)
	at com.utilities_statement/com.utilities_statement.persistent.LoadFlats.loadFlats(LoadFlats.java:36)
	at com.utilities_statement/com.utilities_statement.LoadFlatsTest.testLoadFlats(LoadFlatsTest.java:31)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at junit.framework.TestCase.runTest(TestCase.java:154)
	at junit.framework.TestCase.runBare(TestCase.java:127)
	at junit.framework.TestResult$1.protect(TestResult.java:106)
	at junit.framework.TestResult.runProtected(TestResult.java:124)
	at junit.framework.TestResult.run(TestResult.java:109)
	at junit.framework.TestCase.run(TestCase.java:118)
	at junit.framework.TestSuite.runTest(TestSuite.java:208)
	at junit.framework.TestSuite.run(TestSuite.java:203)
	at org.eclipse.jdt.internal.junit.runner.junit3.JUnit3TestReference.run(JUnit3TestReference.java:124)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:41)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:541)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:763)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:463)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:209)
Caused by: java.lang.IllegalArgumentException: 1
	at com.h2database@1.4.200/org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:166)
	at com.h2database@1.4.200/org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 26 more
2019-11-20 10:08:12 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Feld "year" nicht gefunden
Column "year" not found [42122-200]
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:205)
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:181)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcResultSet.getColumnIndex(JdbcResultSet.java:3169)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcResultSet.get(JdbcResultSet.java:3268)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcResultSet.getDate(JdbcResultSet.java:460)
	at com.utilities_statement/com.utilities_statement.persistent.LoadWaterMeters.loadWaterMeters(LoadWaterMeters.java:26)
	at com.utilities_statement/com.utilities_statement.persistent.LoadFlats.loadFlats(LoadFlats.java:36)
	at com.utilities_statement/com.utilities_statement.client.Main.main(Main.java:27)
2019-11-20 12:21:03 database: flush
org.h2.message.DbException: Allgemeiner Fehler: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/fuhrr/eclipse-workspace/utilities-statement/src/main/resources/database/database.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/fuhrr/eclipse-workspace/utilities-statement/src/main/resources/database/database.mv.db [1.4.200/7]" [50000-200]
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:194)
	at com.h2database@1.4.200/org.h2.message.DbException.convert(DbException.java:347)
	at com.h2database@1.4.200/org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at com.h2database@1.4.200/org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at com.h2database@1.4.200/org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at com.h2database@1.4.200/org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at com.h2database@1.4.200/org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at com.h2database@1.4.200/org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at com.h2database@1.4.200/org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at com.h2database@1.4.200/org.h2.engine.Database.getPageStore(Database.java:2659)
	at com.h2database@1.4.200/org.h2.engine.Database.open(Database.java:675)
	at com.h2database@1.4.200/org.h2.engine.Database.openDatabase(Database.java:307)
	at com.h2database@1.4.200/org.h2.engine.Database.<init>(Database.java:301)
	at com.h2database@1.4.200/org.h2.engine.Engine.openSession(Engine.java:74)
	at com.h2database@1.4.200/org.h2.engine.Engine.openSession(Engine.java:192)
	at com.h2database@1.4.200/org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at com.h2database@1.4.200/org.h2.engine.Engine.createSession(Engine.java:166)
	at com.h2database@1.4.200/org.h2.engine.Engine.createSession(Engine.java:29)
	at com.h2database@1.4.200/org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at com.h2database@1.4.200/org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at com.utilities_statement/com.utilities_statement.persistent.DatabaseController.<init>(DatabaseController.java:13)
	at com.utilities_statement/com.utilities_statement.persistent.LoadFlats.<init>(LoadFlats.java:19)
	at com.utilities_statement/com.utilities_statement.client.Main.main(Main.java:27)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Allgemeiner Fehler: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/fuhrr/eclipse-workspace/utilities-statement/src/main/resources/database/database.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/fuhrr/eclipse-workspace/utilities-statement/src/main/resources/database/database.mv.db [1.4.200/7]" [50000-200]
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 27 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/fuhrr/eclipse-workspace/utilities-statement/src/main/resources/database/database.mv.db [1.4.200/7]
	at com.h2database@1.4.200/org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at com.h2database@1.4.200/org.h2.mvstore.FileStore.open(FileStore.java:172)
	at com.h2database@1.4.200/org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 21 more
2019-11-20 12:21:03 database: flush
org.h2.message.DbException: Allgemeiner Fehler: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/fuhrr/eclipse-workspace/utilities-statement/src/main/resources/database/database.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/fuhrr/eclipse-workspace/utilities-statement/src/main/resources/database/database.mv.db [1.4.200/7]" [50000-200]
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:194)
	at com.h2database@1.4.200/org.h2.message.DbException.convert(DbException.java:347)
	at com.h2database@1.4.200/org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at com.h2database@1.4.200/org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at com.h2database@1.4.200/org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at com.h2database@1.4.200/org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at com.h2database@1.4.200/org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at com.h2database@1.4.200/org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at com.h2database@1.4.200/org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at com.h2database@1.4.200/org.h2.engine.Database.getPageStore(Database.java:2659)
	at com.h2database@1.4.200/org.h2.engine.Database.open(Database.java:675)
	at com.h2database@1.4.200/org.h2.engine.Database.openDatabase(Database.java:307)
	at com.h2database@1.4.200/org.h2.engine.Database.<init>(Database.java:301)
	at com.h2database@1.4.200/org.h2.engine.Engine.openSession(Engine.java:74)
	at com.h2database@1.4.200/org.h2.engine.Engine.openSession(Engine.java:192)
	at com.h2database@1.4.200/org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at com.h2database@1.4.200/org.h2.engine.Engine.createSession(Engine.java:166)
	at com.h2database@1.4.200/org.h2.engine.Engine.createSession(Engine.java:29)
	at com.h2database@1.4.200/org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at com.h2database@1.4.200/org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at com.utilities_statement/com.utilities_statement.persistent.DatabaseController.<init>(DatabaseController.java:13)
	at com.utilities_statement/com.utilities_statement.persistent.LoadBillItems.<init>(LoadBillItems.java:20)
	at com.utilities_statement/com.utilities_statement.client.Main.main(Main.java:29)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Allgemeiner Fehler: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/fuhrr/eclipse-workspace/utilities-statement/src/main/resources/database/database.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/fuhrr/eclipse-workspace/utilities-statement/src/main/resources/database/database.mv.db [1.4.200/7]" [50000-200]
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 27 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/fuhrr/eclipse-workspace/utilities-statement/src/main/resources/database/database.mv.db [1.4.200/7]
	at com.h2database@1.4.200/org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at com.h2database@1.4.200/org.h2.mvstore.FileStore.open(FileStore.java:172)
	at com.h2database@1.4.200/org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 21 more
2019-11-20 12:23:22 database: flush
org.h2.message.DbException: Allgemeiner Fehler: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/fuhrr/eclipse-workspace/utilities-statement/src/main/resources/database/database.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/fuhrr/eclipse-workspace/utilities-statement/src/main/resources/database/database.mv.db [1.4.200/7]" [50000-200]
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:194)
	at com.h2database@1.4.200/org.h2.message.DbException.convert(DbException.java:347)
	at com.h2database@1.4.200/org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at com.h2database@1.4.200/org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at com.h2database@1.4.200/org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at com.h2database@1.4.200/org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at com.h2database@1.4.200/org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at com.h2database@1.4.200/org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at com.h2database@1.4.200/org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at com.h2database@1.4.200/org.h2.engine.Database.getPageStore(Database.java:2659)
	at com.h2database@1.4.200/org.h2.engine.Database.open(Database.java:675)
	at com.h2database@1.4.200/org.h2.engine.Database.openDatabase(Database.java:307)
	at com.h2database@1.4.200/org.h2.engine.Database.<init>(Database.java:301)
	at com.h2database@1.4.200/org.h2.engine.Engine.openSession(Engine.java:74)
	at com.h2database@1.4.200/org.h2.engine.Engine.openSession(Engine.java:192)
	at com.h2database@1.4.200/org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at com.h2database@1.4.200/org.h2.engine.Engine.createSession(Engine.java:166)
	at com.h2database@1.4.200/org.h2.engine.Engine.createSession(Engine.java:29)
	at com.h2database@1.4.200/org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at com.h2database@1.4.200/org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at com.utilities_statement/com.utilities_statement.persistent.DatabaseController.<init>(DatabaseController.java:13)
	at com.utilities_statement/com.utilities_statement.persistent.LoadFlats.<init>(LoadFlats.java:19)
	at com.utilities_statement/com.utilities_statement.client.Main.main(Main.java:27)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Allgemeiner Fehler: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/fuhrr/eclipse-workspace/utilities-statement/src/main/resources/database/database.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/fuhrr/eclipse-workspace/utilities-statement/src/main/resources/database/database.mv.db [1.4.200/7]" [50000-200]
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 27 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/fuhrr/eclipse-workspace/utilities-statement/src/main/resources/database/database.mv.db [1.4.200/7]
	at com.h2database@1.4.200/org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at com.h2database@1.4.200/org.h2.mvstore.FileStore.open(FileStore.java:172)
	at com.h2database@1.4.200/org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 21 more
2019-11-20 12:23:22 database: flush
org.h2.message.DbException: Allgemeiner Fehler: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/fuhrr/eclipse-workspace/utilities-statement/src/main/resources/database/database.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/fuhrr/eclipse-workspace/utilities-statement/src/main/resources/database/database.mv.db [1.4.200/7]" [50000-200]
	at com.h2database@1.4.200/org.h2.message.DbException.get(DbException.java:194)
	at com.h2database@1.4.200/org.h2.message.DbException.convert(DbException.java:347)
	at com.h2database@1.4.200/org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:93)
	at com.h2database@1.4.200/org.h2.mvstore.MVStore.handleException(MVStore.java:2877)
	at com.h2database@1.4.200/org.h2.mvstore.MVStore.panic(MVStore.java:481)
	at com.h2database@1.4.200/org.h2.mvstore.MVStore.<init>(MVStore.java:402)
	at com.h2database@1.4.200/org.h2.mvstore.MVStore$Builder.open(MVStore.java:3579)
	at com.h2database@1.4.200/org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:170)
	at com.h2database@1.4.200/org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:103)
	at com.h2database@1.4.200/org.h2.engine.Database.getPageStore(Database.java:2659)
	at com.h2database@1.4.200/org.h2.engine.Database.open(Database.java:675)
	at com.h2database@1.4.200/org.h2.engine.Database.openDatabase(Database.java:307)
	at com.h2database@1.4.200/org.h2.engine.Database.<init>(Database.java:301)
	at com.h2database@1.4.200/org.h2.engine.Engine.openSession(Engine.java:74)
	at com.h2database@1.4.200/org.h2.engine.Engine.openSession(Engine.java:192)
	at com.h2database@1.4.200/org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at com.h2database@1.4.200/org.h2.engine.Engine.createSession(Engine.java:166)
	at com.h2database@1.4.200/org.h2.engine.Engine.createSession(Engine.java:29)
	at com.h2database@1.4.200/org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at com.h2database@1.4.200/org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at com.h2database@1.4.200/org.h2.Driver.connect(Driver.java:69)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:677)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:228)
	at com.utilities_statement/com.utilities_statement.persistent.DatabaseController.<init>(DatabaseController.java:13)
	at com.utilities_statement/com.utilities_statement.persistent.LoadBillItems.<init>(LoadBillItems.java:20)
	at com.utilities_statement/com.utilities_statement.client.Main.main(Main.java:29)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Allgemeiner Fehler: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/fuhrr/eclipse-workspace/utilities-statement/src/main/resources/database/database.mv.db [1.4.200/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/fuhrr/eclipse-workspace/utilities-statement/src/main/resources/database/database.mv.db [1.4.200/7]" [50000-200]
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at com.h2database@1.4.200/org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	... 27 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/fuhrr/eclipse-workspace/utilities-statement/src/main/resources/database/database.mv.db [1.4.200/7]
	at com.h2database@1.4.200/org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:950)
	at com.h2database@1.4.200/org.h2.mvstore.FileStore.open(FileStore.java:172)
	at com.h2database@1.4.200/org.h2.mvstore.MVStore.<init>(MVStore.java:381)
	... 21 more
